### PR TITLE
MD-503: rewrite plan verification baseline

### PR DESCRIPTION
**Pull request by cchang738**
Make baseline independent of file scan order. @prasadns14 please review.

http://10.10.104.91:8080/job/Framework-Regression/1100/console

### CODE REVIEW

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartorIN_MD-185.e_tsv [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*=.*1993.*>.*29600.*=.*1994.*
     .* 
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartorIN_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiplewithdir_MD-185.e_tsv [hunk 2]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*1993.*>.*29600.*AND.*=.*1994.*>.*29700.* 
     .*
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
\ No newline at end of file
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiplewithdir_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiplewithdir_MD-185.e_tsv [hunk 3]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*1993.*>.*29600.*AND.*=.*1994.*>.*29700.* 
     .*
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
\ No newline at end of file
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiplewithdir_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectStarFromPartition_MD-185.e_tsv [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/ordersjson.*numFiles=2.*columns=\[`\*`\].*files=.*/drill/testdata/partition_pruning/dfs/ordersjson/1994/xae.json,.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json.*
\ No newline at end of file
+.*Filter.*OR.*AND.*=.*1993.*>.*40000.*AND.*=.*1994.*>.*40000.* 
+.*Scan
+.*selectionRoot=
+.*partition_pruning/dfs/ordersjson
+.*numFiles=2
+.*columns=
+.*files=
+.*/drill/testdata/partition_pruning/dfs/ordersjson/1994/xae.json
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectStarFromPartition_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectStarFromPartition_MD-185.e_tsv [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
+.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectStarFromPartition_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartFilter_MD-185.e_tsv [hunk 6]
```diff
@@ -1,3 +1,8 @@
...
-  Filter.*AND.*=.*1993.*>.*40000.*
-    Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/ordersjson.*numFiles=1.*columns=\[`dir0`, `O_TOTALPRICE`, `O_ORDERKEY`, `O_CUSTKEY`, `O_CLERK`, `O_COMMENT`\].*files=.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json.*
+.*Filter.*AND.*=.*1993.*>.*40000
+.*Scan
+.*selectionRoot=
+.*partition_pruning/dfs/ordersjson
+.*numFiles=1
+.*columns=
+.*files=
+.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartFilter_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartor_MD-185.e_tsv [hunk 7]
```diff
@@ -1,3 +1,9 @@
...
-    Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/ordersjson.*numFiles=2.*columns=\[`dir0`, `O_TOTALPRICE`, `O_ORDERKEY`, `O_CUSTKEY`, `O_CLERK`, `O_COMMENT`\].*files=.*/drill/testdata/partition_pruning/dfs/ordersjson/1994/xae.json,.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json.*
+.*Filter.*OR.*AND.*=.*1993.*>.*40000.*=.*1994
+.*Scan
+.*selectionRoot=
+.*partition_pruning/dfs/ordersjson
+.*numFiles=2
+.*columns=
+.*files=
+.*partition_pruning/dfs/ordersjson/1994/xae.json
+.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartor_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartormultiple_MD-185.e_tsv [hunk 8]
```diff
@@ -1,3 +1,9 @@
...
-    Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/ordersjson.*numFiles=2.*columns=\[`dir0`, `O_TOTALPRICE`, `O_ORDERKEY`, `O_CUSTKEY`, `O_CLERK`, `O_COMMENT`\].*files=.*/drill/testdata/partition_pruning/dfs/ordersjson/1994/xae.json,.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json.*
+.*Filter.*condition.*OR.*AND.*=.*1993.*>.*40000.*AND.*1994.*>.*40000
+.*Scan
+.*selectionRoot=
+.*partition_pruning/dfs/ordersjson
+.*numFiles=2
+.*columns=
+.*files=
+.*partition_pruning/dfs/ordersjson/1994/xae.json
+.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartormultiple_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartormultiplewithdir_MD-185.e_tsv [hunk 9]
```diff
@@ -1,3 +1,8 @@
...
-  Filter.*OR.*AND.*=.*1993.*>.*40000.*AND.*=.*1994.*>.*40000.* 
-    Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/ordersjson.*numFiles=2.*columns=\[`dir0`, `O_TOTALPRICE`, `O_ORDERKEY`, `O_CUSTKEY`, `O_CLERK`, `O_COMMENT`\].*files=.*/drill/testdata/partition_pruning/dfs/ordersjson/1994/xae.json,.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json.*
+.*Filter.*OR.*AND.*=.*1993.*>.*40000.*AND.*=.*1994.*>.*40000
+.*Scan
+.*selectionRoot=
+.*partition_pruning/dfs/ordersjson
+.*numFiles=2
+.*columns=
+.*files=.*partition_pruning/dfs/ordersjson/1994/xae.json
+.*/drill/testdata/partition_pruning/dfs/ordersjson/1993/xad.json
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/json/plan/jsonselectpartormultiplewithdir_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiple_MD-185.e_tsv [hunk 10]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*=.*1993.*>.*29600.*AND.*=.*1994.*>.*29700.*
     .*
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
\ No newline at end of file
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiple_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiple_MD-185.e_tsv [hunk 11]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*=.*1993.*>.*29600.*AND.*=.*1994.*>.*29700.*
     .*
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
\ No newline at end of file
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartormultiple_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartor_MD-185.e_tsv [hunk 12]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*=.*1993.*>.*29600.*=.*1994.* 
     .*
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
\ No newline at end of file
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartor_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartor_MD-185.e_tsv [hunk 13]
```diff
@@ -1,4 +1,6 @@
 .*
   Filter.*OR.*AND.*=.*1993.*>.*29600.*=.*1994.* 
     .*
-      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl,.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl.*
\ No newline at end of file
+      Scan.*selectionRoot=.*/drill/testdata/partition_pruning/dfs/lineitempart.*numFiles=2.*columns=\[`dir0`, `columns`\[0\], `columns`\[1\], `columns`\[4\], `columns`\[10\], `columns`\[13\]\].*files=
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1994/lineitemad.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartor_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

File: framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartorIN_MD-185.e_tsv [hunk 14]
```diff
@@ -1,4 +1,6 @@
...
+.*/drill/testdata/partition_pruning/dfs/lineitempart/1993/lineitemac.tbl
```
Comment (prasadns14) [inline framework/resources/Functional/partition_pruning/dfs/csv/plan/csvselectpartorIN_MD-185.e_tsv]: please use * in place of /drill/testdata/ as in other changes

**Comment by cchang738** [comment]
@prasadns14 addressed comments. Please review. Thanks.

**Comment by prasadns14** [comment]
LGTM +1

**Pull request merged**